### PR TITLE
Correct Standalone Tier Page Formatting Broken by `react-intl` Update

### PR DESCRIPTION
**Pull request by SudharakaP**
Resolve https://github.com/opencollective/opencollective/issues/4392

**Screen-shot**

image

### CODE REVIEW

File: components/tier-page/TierContributors.js [hunk 1]
```diff
@@ -61,7 +61,7 @@ export default class TierContributors extends React.Component {
           <H2 mb={3} px={3}>
             <FormattedMessage
               id="TierPage.ContributorsCountGoal"
-              defaultMessage="{userCount, plural, =0 {} one {# individual } other {# individuals }} {both, plural, =0 {} one {and }}{orgCount, plural, =0 {} one {# organization} other {# organizations}} {totalCount, plural, one {has } other {have }} contributed to this goal"
+              defaultMessage="{userCount, plural, =0 {} one {# individual } other {# individuals }} {both, plural, =0 {} one {and } other {}}{orgCount, plural, =0 {} one {# organization} other {# organizations}} {totalCount, plural, one {has } other {have }} contributed to this goal"
```
Comment (Betree) [inline components/tier-page/TierContributors.js:64]: Maybe we can simplify?

```suggestion
              defaultMessage="{userCount, plural, =0 {} one {# individual } other {# individuals }} {both, plural, =0 {} other {and }}{orgCount, plural, =0 {} one {# organization} other {# organizations}} {totalCount, plural, one {has } other {have }} contributed to this goal"
```

**Review by Betree — COMMENTED** [review]
Since this also affects `plural`, are there any other entries that might be affected by this?

File: components/tier-page/TierContributors.js (hunk 1 above)
Comment (SudharakaP) [reply components/tier-page/TierContributors.js]: Yes, Done. 👍🏽

**Comment by SudharakaP** [comment]
> Since this also affects `plural`, are there any other entries that might be affected by this?

@Betree : I did a quick search in the project files for plural and I didn't see any other place where the other clause is missing; 😄 🤔 

```
Targets
    Occurrences of 'plural' in Project
Found Occurrences  (43 usages found)
    Usage in string constants  (43 usages found)
        opencollective-frontend  (43 usages found)
              (1 usage found)
                .eslintrc.json  (1 usage found)
                    71 "formatjs/enforce-plural-rules": ["error"],
            components  (12 usages found)
                CollectiveCard.js  (3 usages found)
                    242 defaultMessage="{n, plural, one {backer} other {backers}}"
                    264 defaultMessage="Contributor to {n, plural, one {Collective} other {Collectives}}"
                    286 defaultMessage="Hosting {n, plural, one {Collective} other {Collectives}}"
                CollectivePicker.js  (3 usages found)
                    27 defaultMessage: '{n, plural, one {Collective} other {Collectives}}',
                    31 defaultMessage: '{n, plural, one {Organization} other {Organizations}}',
                    35 defaultMessage: '{n, plural, one {person} other {people}}',
                CreateGiftCardsForm.js  (1 usage found)
                    47 'All Collectives {nbHosts, plural, =0 {} =1 {under the selected Host} other {under the selected Hosts}}',
                CreateGiftCardsSuccess.js  (2 usages found)
                    78 defaultMessage="Your {count, plural, one {gift card has} other {{count} gift cards have}} been created."
                    131 defaultMessage="Your {count, plural, one {gift card has} other {{count} gift cards have}} been sent!"
                PledgedCollectivePage.js  (1 usage found)
                    151 defaultMessage="{orgCount, plural, =0 {} one {# organization} other {# organizations}} {both, plural, =0 {} other { and }} {userCount
                StyledDropzone.js  (1 usage found)
                    147 defaultMessage="Drop {count,plural, one {file} other {files}} here"
                StyledMembershipCard.js  (1 usage found)
                    53 defaultMessage="{count, plural, one {{prettyCount} contributor} other {{prettyCount} contributors}}"
            components/budget  (1 usage found)
                ExpenseBudgetItem.js  (1 usage found)
                    252 defaultMessage="{count, plural, one {# attachment} other {# attachments}}"
            components/collective-page/sections  (4 usages found)
                Contributions.js  (3 usages found)
                    259 defaultMessage="We are fiscally hosting {n, plural, one {this Collective} other {{n} Collectives}}"
                    314 defaultMessage="{n, plural, one {This Collective is} other {These Collectives are}} part of our Organization"
                    320 defaultMessage="{n, plural, one {This Collective is} other {These Collectives are}} connected to us"
                SponsorsAndParticipants.js  (1 usage found)
                    101 defaultMessage="{n} {n, plural, one {person going} other {people going}}"
            components/conversations  (2 usages found)
                ConversationsList.js  (1 usage found)
                    20 defaultMessage: '{n, plural, one {# comment} other {# comments}}',
                FollowersAvatars.js  (1 usage found)
                    15 defaultMessage: '{usersList} and {count, plural, one {one other} other {# others}}',
            components/discover  (2 usages found)
                DiscoverCollectiveCard.js  (1 usage found)
                    25 defaultMessage="{count, plural, one {contributor {prettyCount} } other {Financial contributors {prettyCount} }}"
                PledgedCollectiveCard.js  (1 usage found)
                    196 defaultMessage="{n, plural, one {backer so far} other {backers so far}}"
            components/edit-collective  (1 usage found)
                EditPaymentMethod.js  (1 usage found)
                    110 defaultMessage="{n} active {n, plural, one {recurring financial contribution} other {recurring financial contributions}}"
            components/edit-collective/sections  (1 usage found)
                PaymentReceipts.js  (1 usage found)
                    130 defaultMessage="{n, plural, one {Transaction} other {Transactions}}"
            components/expenses  (1 usage found)
                ExpenseFormItems.js  (1 usage found)
                    105 defaultMessage="{count, plural, one {The file} other {# files}} failed to upload"
            components/host-dashboard  (1 usage found)
                HostAdminCollectiveCard.js  (1 usage found)
                    78 defaultMessage="<bold>{count}</bold> {count, plural, one {financial contributor} other {financial contributors} }"
            components/tier-page  (1 usage found)
                TierContributors.js  (1 usage found)
                    64 defaultMessage="{userCount, plural, =0 {} one {# individual } other {# individuals }} {both, plural, =0 {} one {and } other {}}{orgCount
            components/updates  (5 usages found)
                UpdateAudienceBreakdown.js  (5 usages found)
                    11 defaultMessage: '{count} {count, plural, one {individual} other {individuals}}',
                    15 defaultMessage: 'The admins of {count} {count, plural, one {organization} other {organizations}}',
                    19 defaultMessage: 'The admins of {count} {count, plural, one {collective} other {collectives}}',
                    23 defaultMessage: 'The admins of {count} {count, plural, one {hosted account} other {hosted accounts}}',
                    27 defaultMessage: '{count} {count, plural, one {core contributor} other {core contributors}}',
            lib/hooks  (1 usage found)
                useImageUploader.js  (1 usage found)
                    14 defaultMessage: 'The following {count, plural, one {file is} other {files are}} not valid: {files}',
            lib/i18n  (4 usages found)
                collective-type.js  (4 usages found)
                    8 defaultMessage: '{count, plural, one {Organization} other {Organizations}}',
                    12 defaultMessage: '{count, plural, one {User} other {Users}}',
                    16 defaultMessage: '{count, plural, one {Collective} other {Collectives}}',
                    20 defaultMessage: '{count, plural, one {Event} other {Events}}',
            pages  (6 usages found)
                banner-iframe.js  (2 usages found)
                    371 '{n} {n, plural, one {organization is} other {organizations are}} supporting {collective}'
                    405 defaultMessage={'{n} {n, plural, one {individual is} other {individuals are}} supporting {collective}'}
                createPledge.js  (1 usage found)
                    549 'by {orgCount, plural, =0 {} one {# sponsor} other {# sponsors}} {both, plural, =0 {} other { and }} {userCount, plural, =0 {} one {# backer } other {# backers }}'
                search.js  (3 usages found)
                    73 defaultMessage: '{count, plural, one {Collective} other {Collectives}}',
                    77 defaultMessage: '{count, plural, one {Event} other {Events}}',
                    81 defaultMessage: '{count, plural, one {Organization} other {Organizations}}',

```

**Review by Betree — APPROVED** [review]
Great then :raised_hands:

**Pull request merged**